src/Illuminate/Encryption/KeyDecoder.php

```diff
@@ -0,0 +1,32 @@
+<?php
+
+
+namespace Illuminate\Encryption;
+
+
+trait KeyDecoder
+{
+    public function decode($encodedKey)
+    {
+        $possibleDecoders = [
+            'base64' => function ($encoded) {
+                return base64_decode($encoded);
+            },
+        ];
+
+        $splitValue = explode(":", $encodedKey);
+
+        if (count($splitValue) < 2) {
+            return null;
```

Comment:
Should we return null if the `count($splitValue) < 2`. Wouldn't we just return the `$encoded` key value since that'd mean that they actually didn't specify a decoder.